### DISCUSSION TITLE
Microsteps and current settings

### DISCUSSION THREAD

**Discussion by jaymcd** [OpenNeptune3D/OpenNept4une#167] [General]
Has anyone adjusted their microsteps to more than 16? I saw a config from Tom's Basement (Youtube 3D Contributor) that had soem "optimized" settings for the original firmware and was wondering if there is any benefit to reducing the motor driver current, and  increasing the microsteps.

**Comment by halfmanbear** [comment]
I run 32 microsteps on the X and Y but I wouldn't lower the current.

If you scroll down this section to find the example `user_settings.cfg` you'll see the required changes to run 32 microsteps properly. It will increase the digital stepper noise significantly but won't loose too much torque.

https://github.com/OpenNeptune3D/OpenNept4une/wiki/Updating-your-printer.cfg#keeping-your-modified-settings
